src/main/java/com/softserveinc/ita/pageobjects/HomePage.java

```diff
@@ -29,4 +36,19 @@ public ProductPage openLastProductOfSection(String sectionName) {
 
         return new ProductPage();
     }
+
+    public HomePage openSocialMediaPage(String mediaName) {
+        $x(format("//a[contains(@class, 'socials__link') and @title = '%s']", mediaName)).click();
+
+        return this;
+    }
+
+    public boolean isSocialMediaCorrect(String mediaName) {
+        String expectedUrl = SocialMedia.valueOf(mediaName.toUpperCase()).getSocialMediaLink();
+        Selenide.switchTo().window(1);
```

Comment:
create util class and name it something like BrowserTabHelper and move there these lines and create separate methods such as
void switchToTab(int tabNumber){switchTo().window(tabNumber);}
String getCurrentUrl(){return url();}
void closeTab(){closeWindow();}


and in a place where you will need these methods, you will need to call it through a variable or just new BrowserTabHelper().getCurrentUrl();

after closing tab you probably need to switch to first tab